cpp/kiss_icp/core/VoxelHashMap.cpp

```diff
@@ -138,6 +138,7 @@ std::vector<Eigen::Vector3d> VoxelHashMap::Pointcloud() const {
     std::vector<Eigen::Vector3d> points;
     points.reserve(max_points_per_voxel_ * map_.size());
     for (const auto &[voxel, voxel_block] : map_) {
+        (void)voxel;
         std::for_each(voxel_block.points.cbegin(), voxel_block.points.cend(),
```

Comment:
Indeed, you are right, I think it was leftovers from where I was trying to parallelize that region somehow (Usually I use std::for_each everywhere because of this)